src/client/driver/driver.js

```diff
@@ -1063,10 +1064,14 @@ export default class Driver extends serviceUtils.EventEmitter {
         const wnd      = this.parentWindowDriverLink?.getTopOpenedWindow() || window;
         const response = await sendMessageToDriver(new GetWindowsMessage(), wnd, WAIT_FOR_WINDOW_DRIVER_RESPONSE_TIMEOUT, CannotSwitchToWindowError);
 
-        this._onReady(new DriverStatus({
+        // NOTE: we need to use stringify/parse combo to create clear JS array
+        // otherwise the tests with fail in IE11
+        const status = new DriverStatus({
```

Comment:
I've found a 'DataCloneError' error in IE in the hammerhead's transport (https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/transport/index.ts#L108) subsystem.

I guess it can occur due to the strange type of the 'sendMessageToDriver' function result. It should be an array, it has an empty prototype.

![image](https://user-images.githubusercontent.com/4133518/88798065-fac4c100-d1ac-11ea-917b-b5c519f85ca5.png)

Need to debug the rollup service code,  that realizes the support of async/await functionality to choose the most suitable fix for this problem.